aiogoogle/client.py

```diff
@@ -141,26 +141,35 @@ async def list_api(self, name, preferred=None, fields=None):
         )
         return await self.as_anon(request)
 
-    async def discover(self, api_name, api_version=None, validate=False):
-        """ 
+    async def discover(self, api_name, api_version=None, validate=False, *, api_endpoint_url_fmt=None):
```

Comment:
Why the extra spread here?

Reply:
This makes `api_endpoint_url_fmt` a [keyword-only](https://docs.python.org/3/glossary.html#term-parameter) parameter.

It's just a habit I do sometimes if I'm worried that passing the param in without specifying what param it is via the keyword would be ambiguous; I can remove it though, since it's really not adding any value.

Reply:
Cool! didn't know about it. Hard to google an explanation for it as well :)
I'm indifferent about it tbh, your choice.